magpie/magpieadapter/__init__.py

```diff
@@ -0,0 +1,72 @@
+import logging
+import os
+import sys
+
+# TODO Is the following block really useful?
```

Comment:
this is changed in the branch I'm currently working on (api+unittests)
these imports are required to allow finding proper modules/files no matter where the $pwd is set

Reply:
👍